.eslintrc.js

```diff
@@ -115,6 +115,12 @@ module.exports = {
         ],
         '@typescript-eslint/restrict-plus-operands': ['warn'],
         '@typescript-eslint/ban-ts-comment': ['warn'],
+        '@typescript-eslint/no-empty-interface': [
```

Comment:
Why do we need this?

Reply:
This is needed for the declaration file, [styled.d.ts](https://github.com/equinor/design-system/blob/746f719b3267e173bf5f4f18e5719fff840cae30/libraries/core-react/src/components/styled.d.ts) where we extend the type for `styled-components` `ThemeProvider`. https://styled-components.com/docs/api#create-a-declarations-file